classes/class-wc-connect-service-settings-store.php

```diff
@@ -110,6 +110,40 @@ public function can_user_manage_payment_methods() {
 				( is_a( $master_user, 'WP_User' ) && $current_user->ID === $master_user->ID );
 		}
 
+		/**
+		 * Gets the store's location settings.
+		 *
+		 * @return array
+		 */
+		public function get_endshipper_address() {
```

Comment:
Let's stay consistent in nomenclature, i.e. let's call this either "end shipper" or "endshipper". I think "end shipper" is a valid term so let's replace this with:

```suggestion
		public function get_end_shipper_address() {
```

Reply:
Yes, let's use "end shipper". Just want to confirm, in WCC Server, I have created a new table for end shippper with the name : `wcc_endshipper`. Should we change it to `wcc_end_shipper` too?

Reply:
I think that'd be nice. That way we don't have to wonder if we separated these words or not.

Reply:
Thanks for the advice, I have separated `endshipper` word into `end_shipper` or `endShipper` on some variables.
I have done this on both wcc services and wcc server.